### PR TITLE
#163149785 Update ReadMe

### PR DESCRIPTION
**Pull request by Nyakaru**
## What does this PR do?
Update README to capture necessary setup instructions and add links for mock ups

### Description of task to be completed
- update README on how to manually setup and test the application
- add links to mock up designs of the application

### What are the relevant pivotal tracker stories?
[#162948978](https://www.pivotaltracker.com/story/show/162948978)
[#163149785](https://www.pivotaltracker.com/story/show/163149785)

#### Landing page
screenshot 2019-01-17 at 13 54 05

#### Home page
screenshot 2019-01-17 at 13 54 36

#### Profile page 
screenshot 2019-01-17 at 13 54 54

#### Create article
screenshot 2019-01-17 at 13 55 40

#### Sign Up
screenshot 2019-01-17 at 13 56 00

#### Login
screenshot 2019-01-17 at 13 56 22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +15,38 @@ by leveraging the modern web.
...
+1. Activate your virtual environment: `source venv/bin/activate`
+1. Clone this repo with the following command:
+`git clone https://github.com/andela/ah-fulldeck.git`
+1. Cd into the ah-fulldeck directory:
+`cd ah-fulldeck`
+1. Install the requirements by running the command:
+`pip install requirements.txt`
+1. Create a database:
+`create database fulldeck`
+1. Export the environment settings as follows:
```
Comment (eduhmik) [inline README.md]: Include the email configurations too... you can check my PR for all the environment settings

**Comment by Hesbon5600** [comment]
You could include the heroku link in the README

File: README.md [hunk 2]
```diff
@@ -14,8 +15,38 @@ by leveraging the modern web.
...
+[Full deck link](https://wireframepro.mockflow.com/view/M15c07edf891b45f0dfaa1f5cc85c68c61547577768098#/page/D31452032c99b79d208bf34f194933048)
+
+2. ### Homepage, Landing page and Profile
+ [Full deck link](https://wireframepro.mockflow.com/view/M724c1af1281229c8914e2f1e3b9e2a201547621799851)
+
+**How should this be manually tested?**
+
+1. Install pip first using this command `sudo apt-get install python3-pip`
+1. Then install virtualenv using pip3 `sudo pip3 install virtualenv `
+1. Now create a virtual environment `virtualenv venv `
```
Comment (eduhmik) [inline README.md]: Add a windows option on how to create a virtualenv and how to activate it

Comment (Nyakaru) [reply README.md]: Kindly go through the ReadMe again..it has a part on how to create and activate virtualenv

File: README.md (hunk 1 above)
Comment (Nyakaru) [reply README.md]: Thanks. Working on it.

**Comment by Nyakaru** [comment]
> You could include the heroku link in the README

Thanks. Working on it.

**Comment by charitymarani** [comment]
@Nyakaru Indicate titles for each of your screenshots so that a person reading your PR can easily know what page they are looking at

**Comment by charitymarani** [comment]
And also you say you have added links to mockups yet i can't see any links I just see screenshots. Did you mean screenshots or did you forget to add the links you are talking about?

**Comment by Nyakaru** [comment]
> And also you say you have added links to mockups yet i can't see any links I just see screenshots. Did you mean screenshots or did you forget to add the links you are talking about?

The links are in the readMe

**Comment by Nyakaru** [comment]
> And also you say you have added links to mockups yet i can't see any links I just see screenshots. Did you mean screenshots or did you forget to add the links you are talking about?

Working on it

File: README.md [hunk 3]
```diff
@@ -14,8 +15,62 @@ by leveraging the modern web.
 ## API Spec
 The preferred JSON object to be returned by the API should be structured as follows:
 
-### Users (for authentication)
+## Links to the Mock up Design
+1. ### Sign Up, Log In and Create Article
+    [Full deck link](https://wireframepro.mockflow.com/view/M15c07edf891b45f0dfaa1f5cc85c68c61547577768098#/page/D31452032c99b79d208bf34f194933048)
+
+2. ### Homepage, Landing page and Profile
+   [Full deck link](https://wireframepro.mockflow.com/view/M724c1af1281229c8914e2f1e3b9e2a201547621799851)
```
Comment (kzyangiro) [inline README.md]: 1. Mockup links number 1 and 2, make them the actual links, and not bold. Remove **Full deck link** not ideal

File: README.md [hunk 4]
```diff
@@ -14,8 +15,62 @@ by leveraging the modern web.
...
+
+
+## Hosting
+Staging-https://ah-fulldeck-staging.herokuapp.com/
+
+
+## How should this be manually tested?
+
```
Comment (kzyangiro) [inline README.md:38]: 2. **How should this be manually tested**? in README? Really?

File: README.md [hunk 5]
```diff
@@ -14,8 +15,62 @@ by leveraging the modern web.
...
+1. Create a database:
+  - In the terminal run the following after installing postgres and setting up:  
+  `psql`
+  
+    `create DATABASE <name_of_your_database>`
+
+1. Export the environment settings as follows:  
+`export DATABASE="<name of your database>"`  
+   
+   `export USER="<your postgres username>"`
```
Comment (kzyangiro) [inline README.md:61]: 3. Number 1 to 8 not well spaced, And how does number 1 come after number 8? Number 2 (The second number 2) seems well organised, use the same structure for all other instructions

File: README.md [hunk 6]
```diff
@@ -14,8 +15,62 @@ by leveraging the modern web.
...
+   `export EMAIL_SENDER="<your email sender>"`
+   
+   `export EMAIL_HOST="smtp.<hostname>.com"`
+   
+   `export EMAIL_HOST_USER="<email username>"`
+
+   `export EMAIL_HOST_PASSWORD="<email host password>"`
+   
+   `export EMAIL_PORT=<host port>`
+1. Migrate the models to the db:
```
Comment (kzyangiro) [inline README.md]: 4. Number 1 (Second number 1) Indicate that user need to create a .env, and add those exports. and indicate the actual value of the email exports.

Comment (kzyangiro) [inline README.md]: Migrate models to DB? Is that correct? Emmm, I don't think so.

**Review by kzyangiro — CHANGES_REQUESTED** [review]
--> Try clicking on **view file** to see the preview of the README page its not very okey.

File: README.md (hunk 5 above)
Comment (Hesbon5600) [reply README.md:61]: @kzyGit Markdown allows you to just write 1. ....1. ...1. And it'll format it to 1. .... 2. ...3. ...

File: README.md (hunk 6 above)
Comment (Nyakaru) [reply README.md]: I have changed that to migrate the database

Comment (Nyakaru) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (Nyakaru) [reply README.md:61]: @Hesbon5600 has clarified that. Its markdown styling. If you check the Readme its well numbered.

File: README.md (hunk 4 above)
Comment (Nyakaru) [reply README.md:38]: Worked on

File: README.md (hunk 3 above)
Comment (Nyakaru) [reply README.md]: Done

**Comment by Nyakaru** [comment]
> --> Try clicking on **view file** to see the preview of the README page its not very okey.

Okay this happens when you force push to Github. It overwrites the changes on the **view file**

**Pull request merged**